client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -59,27 +59,81 @@ export const filterBusinessExtensions = ( extensionInstallationOptions ) => {
 	);
 };
 
+const timeBoxes = [
+	{ name: '0-2s', max: 2 },
+	{ name: '3-4s', max: 5 },
+	{ name: '5-9s', max: 10 },
+	{ name: '10-15s', max: 15 },
+	{ name: '>15s' },
+];
+function getTimebox( timeInMs ) {
+	for ( const timeBox of timeBoxes ) {
+		if ( ! timeBox.max ) {
+			return timeBox.name;
+		}
+		if ( timeInMs < timeBox.max * 1000 ) {
+			return timeBox.name;
+		}
+	}
+}
+
+function getExtensionKey( fieldKey ) {
+	const key =
+		fieldKey === 'woocommerce-payments'
+			? 'wcpay'
+			: `${ fieldKey.replace( /-/g, '_' ) }`.split( ':', 1 )[ 0 ];
```

Comment:
We're doing this enough that it may warrant a shared util or some sanitization on the backend even before hitting components, but not a blocker for this PR.

Reply:
Done here, thanks for the suggestion: https://github.com/woocommerce/woocommerce-admin/pull/8042/commits/cbf184d5acaf52e8df59d0b0b2658c34c2267825